docs/props/render-engines.md

```diff
@@ -0,0 +1,51 @@
+# `renderEngines` property
+
+## Overview
+
+The `renderEngines` property is required and contains render engines for component.
+
+Render engine is pure React component which task is render content of specific type of document to html.
+
+## Types
+
+For types of render engines, see [file](https://github.com/kyma-incubator/documentation-component/blob/master/packages/documentation-component/src/interfaces/RenderEngine.ts).
+
+## Passing globally options
+
+`renderEngines` property allows to define options for single render engine. Instead of passing a single one, pass object with `renderEngine` field (with appropriate render engine as value) and second `options` field. See example.
```

Comment:
```md
The `renderEngines` property allows also you to define options for a single render engine. Instead of passing a single one, pass an object with the **renderEngine** field with the appropriate render engine as value, and add the **options** field. 
```